action.yml

```diff
@@ -16,6 +16,9 @@ inputs:
   version-type:
     description: Bumps the package version based on value (e.g. patch)
     default: patch
+  base-brach:
+    description: Pull request base branch.
+    default: master
```

Comment:
Not all repos have the default branch set to `master` (e.g. `main`). Before this change, `hub` would [detect it](https://hub.github.com/hub.1.html). After this change, this will break for users with a default branch that isn't `master`.

The best bet is to probably leave this blank as default, and decide what to do in `entrypoint.sh`.